tests/pact/parsing.repl

```diff
@@ -50,3 +55,8 @@
  "list equivalence with commas or without"
  [1 "2" true (+ 3 4)]
  [1, "2", true, (+ 3 4)])
+
+(expect
```

Comment:
I would test cases individually.

Reply:
I found a more clever solution :^)

Reply:
This is fine but in general avoid overly terse tests going forward. A best practice for tests is verbosity for ease of comprehension for the maintainer.